modules/c++/math.linear/include/math/linear/MatrixMxN.h

```diff
@@ -1189,8 +1189,26 @@ class MatrixMxN
         return multiply(mx);
     }
     
-
-
+    /*!
+     *  Negation operator;
+     *
+     *  \code
+           B = -A;
+     *  \endcode
+     *
+     */
+    Like_T operator-() const
+    {
+        Like_T mneg;
+        for (size_t i = 0; i < _MD; ++i)
```

Comment:
Avoid single character variable names (motivation is they're harder to do find/replaces on, though granted any decent IDE has a 'whole word' option).  So, these would change to `ii` and `jj`.